FinSearchUnified/Resources/config.xml

```diff
@@ -27,6 +27,18 @@
             <description lang="de">Aktiviert die FINDOLOGIC Suche für Kategorieseiten.</description>
             <description lang="en">Activate the FINDOLOGIC search provider for category pages.</description>
         </element>
+        <element type="combo">
```

Comment:
@zaifastafa I click-tested this on my own with Shopware 5.6 and it seems to work fine. On Shopware 5.2 it also works, but I am not able to select sub-categories.

![image](https://user-images.githubusercontent.com/20555112/72810282-850bf580-3c5d-11ea-9471-0df50854b3fb.png)

The moment I type anything that does not match any of those two categories (e.g. a sub category), it does not work.
Please ensure that you can also select sub-categories.